bridge-api/definitions/enums/ui_hint.yml

```diff
@@ -17,11 +19,14 @@ description: |
     |textfield|A single line text field. Note that if there are set values for this textfield, but the user can essentially enter an "other" value as well, then the combobox hint with a set of enumerated values should be used.||
     |timepicker|An input for entering a time with no reference to the date of the time.||
     |toggle|A switch or toggle control to select a boolean state||
+    |weight|A control for entering a weight value||
```

Comment:
There's an infant toggle in the data, I would mention that, e.g. "A control for entering an adult or infant weight value in a specified unit of measurement."